chrome_manifest_v3/views/popupView.js

```
@@ -380,7 +381,7 @@ Gmail2Trello.PopupView.prototype.updateBody = function (data = {}) {
 
 Gmail2Trello.PopupView.prototype.bindEvents = function () {
   // bind events
-  var self = this;
+  let self = this;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'self' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)